views/members-view.js

```diff
@@ -17,22 +17,49 @@ var app = app || {};
 
     $("#member-display").append(
       `
-      <h2>${member.firstName} ${member.lastName}</h2>
+      <div id="${member._id}">
+      <h2><span name="firstName" value=${member.firstName}>${member.firstName}</span>&nbsp;<span name="lastName" value=${member.lastName}>${member.lastName}</span></h2>
       <ul>
-        <li>Skills: ${skills}</li>
-        <li>LinkedIn URL: ${member.linkedInUrl}</li>
-        <li>GitHub URL: ${member.gitHubUrl}</li>
-        <li>Profile URL: ${member.profileUrl}</li>
-        <li>Email: ${member.email}</li>
+        <li name="skills" value=${member.skills}>Skills: ${skills}</li>
+        <li name="linkedInUrl" value=${member.linkedInUrl}>LinkedIn URL: ${member.linkedInUrl}</li>
+        <li name="gitHubUrl" value="${member.gitHubUrl}">GitHub URL: ${member.gitHubUrl}</li>
+        <li name="profileUrl"value="${member.profileUrl}">Profile URL: ${member.profileUrl}</li>
+        <li name="email" value="${member.email}" >Email: ${member.email}</li>
       </ul>
+      <button name="delete" value="${member._id}">Delete</button>
```

Comment:
I think these options should go in an admin page instead so making changes is not so out to the public? But I suppose if we add a login that just removes these items for non logged in people, it's good also.

Reply:
My thoughts were to just have a single page app with proper claims that controls which functions are available.